.github/workflows/ci.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       matrix:
         python-version: ["3.10"]
     env:
+      CPUCOUNT: 1
```

Comment:
Est-ce qu’on pourrait ne forcer cette valeur que pour la tâche où c’est nécessaire :

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsenv

Reply:
Je dis peut être une bétise, mais vu qu'on est dans la section build, est-ce que ce n'est pas déjà le cas ?

Reply:
Ma suggestion est de ne forcer `CPUCOUNT=1` que pour l’étape `pip install` et pas pour tout le build. L’intérêt immédiat étant d’éviter `CPUCOUNT=1` pour les tests, lint, etc.

Reply:
Ca semble en effet assez facile à faire, bonne idée !

Pour info, plus de contexte sur cette valeur:
- faire CPUCOUNT=1 est un hack qu'on est obligé de faire parce que Clever n'a pas reussi à fix ses builds sur leurs VM. Ils ont une corruprion de FS sur laquelle on a eu une investigation longue et le resultat de leur coté etait :shrug: 
- CPUCOUNT=1 est une variable a priori uniquement prise en compte par la compilation de uwsgi et rien d'autre donc normalement pas d'impact ailleurs. J'ai cherché CPUCOUNT dans github, greppé dans nos dépendances, rien trouvé.